Networking/Sources/Networking/Requests/MealDetails/MealDetailsResponse.swift

```diff
@@ -0,0 +1,57 @@
+// MARK: - MealDetailsResponse
+public struct MealDetailsResponse: Decodable {
```

Comment:
```suggestion
struct MealDetailsResponse: Decodable {
```
No need to make this public, just return a list of meals.